target/linux/ath79/dts/qca9531_comfast_cf-wr752ac.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-wr752ac", "qca,qca9531";
+	model = "COMFAST CF-WR752AC";
```

Comment:
If this is a v1, please include that in all the names:
```
	compatible = "comfast,cf-wr752ac-v1", "qca,qca9531";
	model = "COMFAST CF-WR752AC v1";
```
Same for file names, imake Makefile, board.d etc.

Reply:
I didn't use v1 since there is no mention on the device itself, but since it seems to be the good  usance, no problem.